migrate_pim/upgrade/upgrade_from_40_to_50.rst

```diff
@@ -0,0 +1,263 @@
+Upgrade from 4.0 to 5.0
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Use this documentation to upgrade projects based on Akeneo PIM Community Edition or Enterprise Edition 4.0 to 5.0.
+
+Disclaimer
+**********
+
+Make sure your production database is backed-up before performing the data migration.
+The queue daemon(s) must be stopped as well.
+
+Requirements
+************
+
+Updated System components
+-------------------------
+
+You have to make sure your system components are updated to the version required for Akeneo PIM:
+ - PHP 7.4
+ - MySQL 8.0
+ - Elasticsearch 7.10
+
+.. note::
+    Elasticsearch supports in-place update: Elasticsearch 7.10 will be able to use indexes created
+    by Elasticsearch 7.5.
+
+    So there's no need to export and reimport data for this system.
+
+
+Updated System dependencies
+---------------------------
+
+Community Edition
+^^^^^^^^^^^^^^^^^
+You can download the composer.json file directly from the Github repository: https://raw.githubusercontent.com/akeneo/pim-community-standard/5.0/composer.json
+
+Enterprise Edition
```

Comment:
Suppressed: documented in sytem requirements
@mr-prud

Reply:
yes!